chat-client/src/client/messager.ts

```diff
@@ -61,16 +61,17 @@ export class Messager {
     }
 
     onSendToPrompt = (params: SendToPromptParams, tabId: string): void => {
-        this.chatApi.tabIdReceived({
-            eventId: params.eventId,
-            tabId: tabId,
-        })
+        this.chatApi.telemetry({ ...params, tabId })
```

Comment:
Shouldn't telemetry have `name` field attached to it?

Reply:
Telemetry changes are out of scope for this PR. I will publish new PR for telemetry. See https://github.com/aws/language-servers/pull/290/commits/ea854cef46a2eeb413513a9ba54a2a53a4b6671b for current status on that